assets/app/index.html

```diff
@@ -243,6 +244,7 @@
         <script src="scripts/directives/editRequestLimit.js"></script>
         <script src="scripts/directives/editProbe.js"></script>
         <script src="scripts/directives/editCommand.js"></script>
+        <script src="scripts/directives/runDockerImage.js"></script>
```

Comment:
maybe `fromDockerImage` ? I've named the `From File` tab as `fromFile`. Would be good to align on one format :)